packages/xo-server/src/xo-mixins/api.js

```diff
@@ -255,7 +255,7 @@ export default class Api {
         .slice(2),
       userId,
       userName,
-      userIp: session.get('user_ip'),
+      userIp: session.get('user_ip', undefined),
```

Comment:
How major is this issue?

Should we release a patch?

Reply:
It's very problematic for scheduled API jobs. I don't know if users use the jobs interface. If it is we need to release a patch.

Reply:
Is this really normal for this entry to not be defined for scheduled jobs?

Reply:
IMHO, technically yes, functionally  no, as mentioned in the PR description, this case has not been handle by the audit i've added it to audit step 3.

Reply:
IMHO a patch release is unnecessary, let's just merge this in master, we'll do a patch if necessary.